string dst_robot
string topic
adhoc_communication/RzvPoint msg
---
uint8 status
